.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v2
+
+      - name: Install correct version of Node
+        shell: bash -l {0}
+        run: nvm install
+
+      - name: Install correct version of Yarn
+        run: yarn policies set-version $(cat .yvmrc)
```

Comment:
Yarn has utilities included to [install and enforce which version of Yarn you use](https://classic.yarnpkg.com/en/docs/cli/policies/). I think `yvm` is more convenient for local dev, but `policies` works well here. Under the hood, it's installing Yarn and saving a bundled copy of it in a local directory. If we based our workflow around this we'd commit that copy and Yarn would know to use it automatically. In this context, it just makes it easy to quickly install the correct version of Yarn.

Reply:
Similar comment here regarding Yarn -- although I'm not sure what version gets packaged through `setup-node` or the base image.

Reply:
I'd be comfortable relying on whatever crops up by default for CI builds. It's been fairly stable with respect to installing packages from npm, which is how users will typically consume this library. Pinning the version used to build the published artifact is more valuable than the version used to install it.